packages/eslint-config-react-app/package.json

```diff
@@ -17,23 +17,23 @@
     "jest.js"
   ],
   "peerDependencies": {
-    "eslint": "^7.30.0"
+    "eslint": "^8.0.0"
```

Comment:
We've got another PR to update ESLint so can you leave this as `7.32.0` in this PR?

Reply:
I have moved eslint back down to 7.32.0, I also reset the plugins as some are eslint 8 only and are updated in #11375 anyway, if you need anything else let me know